js/fc.js

```diff
@@ -39,6 +39,7 @@ var FAILSAFE_CONFIG;
 var RXFAIL_CONFIG;
 var PID_ADVANCED_CONFIG;
 var FILTER_CONFIG;
+var SPECIAL_PARAMETERS;
```

Comment:
We decided we would not put this back in only to support 2.9.0, and instead release a 2.9.1 maintenance release that includes the new MSP protocol.